.github/workflows/go-32bit.yml

```diff
@@ -0,0 +1,83 @@
+name: CI for 32-bit
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: "*"
+
+jobs:
+
+  build:
+    name: ${{ matrix.os }} - Go ${{ matrix.go_version }}
+    runs-on: ubuntu-latest
```

Comment:
How to run on OS 32-bit

Reply:
in another file 
<img width="385" alt="image" src="https://user-images.githubusercontent.com/43714056/183413237-f0c2dc1b-3a70-45a0-a5c0-183a05bc7225.png">

follow is the ci logs in github.
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/43714056/183414101-a2a1af99-8402-476a-b166-e781fe8c688f.png">




you also can run it manual:
1、download the install script https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh
2、change the arch param  from $(uname_arch) to 386, use command `  sed -i 's/ARCH=$(uname_arch)/ARCH=386/g' `install.sh
<img width="491" alt="image" src="https://user-images.githubusercontent.com/43714056/183413534-41045afe-c852-48e7-9752-74d333fc7727.png">
3、then execute the install.sh  by `. install.sh -b $(go env GOPATH)/bin v1.27.0`
4、golangci-lint run --timeout=10m -v --disable-all --enable=govet --enable=staticcheck --enable=ineffassign --enable=misspell

Reply:
there is no os image of 32-bit for running

Reply:
there's no os image of 32-bit for running
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/43714056/183416182-3178e493-4246-4c1c-9d0e-548fa1d6cc25.png">